tests/test_validation.py

```diff
@@ -1083,3 +1054,901 @@ def test_login(self, dmge: DataModelGraphExplorer) -> None:
+                [],
+                [],
+            )
+            assert va_obj.cross_validation(val_rule, Series(["A", "B", "C", "D"])) == (
+                [],
+                [],
+            )
+
+    def test_cross_validation_match_none_set_rules_errors(
+        self, va_obj: ValidateAttribute, cross_val_df1: DataFrame
+    ):
+        """Tests for cross manifest validation for matchNone set rules"""
+        val_rule = "matchNone Patient.PatientID set error"
+
+        with patch.object(
+            schematic.models.validate_attribute.ValidateAttribute,
+            "_get_target_manifest_dataframes",
+            return_value={"syn1": cross_val_df1},
+        ):
+            errors, _ = va_obj.cross_validation(
+                val_rule,
+                Series(["A", "B", "C"], index=[0, 1, 2], name="PatientID"),
+            )
+            assert len(errors) == 1
+
+            errors, _ = va_obj.cross_validation(
+                val_rule,
+                Series(["A", "B", "C", "C"], index=[0, 1, 2, 3], name="PatientID"),
+            )
+            assert len(errors) == 1
+
+    def test_cross_validation_value_match_atleast_one_rules_passing(
+        self, va_obj: ValidateAttribute, cross_val_df1: DataFrame
+    ):
+        """Tests for cross manifest validation for matchAtLeastOne value rules"""
+        val_rule = "matchAtLeastOne Patient.PatientID value error"
+
+        with patch.object(
+            schematic.models.validate_attribute.ValidateAttribute,
+            "_get_target_manifest_dataframes",
+            return_value={"syn1": cross_val_df1},
+        ):
+            assert va_obj.cross_validation(val_rule, Series([])) == ([], [])
+            assert va_obj.cross_validation(val_rule, Series(["A"])) == ([], [])
+            assert va_obj.cross_validation(val_rule, Series(["A", "A"])) == ([], [])
+            assert va_obj.cross_validation(val_rule, Series(["A", "B"])) == ([], [])
+            assert va_obj.cross_validation(val_rule, Series(["A", "B", "C"])) == (
+                [],
+                [],
+            )
+            assert va_obj.cross_validation(val_rule, Series(["A", "B", "C", "C"])) == (
+                [],
+                [],
+            )
+
+    def test_cross_validation_value_match_atleast_one_rules_errors(
+        self, va_obj: ValidateAttribute, cross_val_df1: DataFrame
+    ):
+        """Tests for cross manifest validation for matchAtLeastOne value rules"""
+        val_rule = "matchAtLeastOne Patient.PatientID value error"
+
+        with patch.object(
+            schematic.models.validate_attribute.ValidateAttribute,
+            "_get_target_manifest_dataframes",
+            return_value={"syn1": cross_val_df1},
+        ):
+            errors, _ = va_obj.cross_validation(
+                val_rule, Series(["D"], index=[0], name="PatientID")
+            )
+            assert len(errors) == 1
+
+    def test_cross_validation_match_exactly_ne_value_rules_passing(
```

Comment:
there's a typo in the function right? shouldn't it be: `test_cross_validation_match_exactly_one_value_rules_passing`

Reply:
Yep!